helpers.rb

```diff
@@ -6,8 +6,17 @@ def wait_for(search)
   @browser.find_element(search)
 end
 
+# columns go from: A..Z, AA..AZ, BA..BZ,
 def char_to_col_index(char)
   return char if char.is_a? Numeric
   return char if char == nil
-  char.upcase.ord - 65
+  pos = 0
+  chars = char.split('')
+  chars.each do |char|
+    # need to be able to calculate columns after AZ
+    # currently only accurate up to AZ
+    pos = char.upcase.ord - 65
+  end
+  binding.pry
+  pos = pos + 26 * (chars.length - 1)
```

Comment:
@knutsenm this custom method only supported through 'A', in the next migration we have column 'AA' which returned 0 instead of 26. I'm working on an adaptive method that would be able to translate any column. Still a WIP

Reply:
Base 26, kewl

Reply:
Well, kinda - base 26 with no zero.